www/js/survey/enketo/enketo-add-note-button.js

```diff
@@ -147,44 +149,43 @@ angular.module('emission.survey.enketo.add-note-button',
     resultMap[enbs.SINGLE_KEY] = surveyResults;
   }
 
-  enbs.populateInputsAndInferences = function(trip, manualResultMap) {
-    console.log("ENKETO: populating trip,", trip, " with result map", manualResultMap);
-    if (angular.isDefined(trip)) {
-        // initialize addition arrays as empty if they don't already exist
-        trip.trip_addition ||= [], trip.place_addition ||= [];
-        trip.tripAddition ||= [], trip.placeAddition ||= [];
-        enbs.populateManualInputs(trip, trip.nextTrip, enbs.SINGLE_KEY,
+  enbs.populateInputsAndInferences = function(timelineEntry, manualResultMap) {
+    console.log("ENKETO: populating timelineEntry,", timelineEntry, " with result map", manualResultMap);
+    if (angular.isDefined(timelineEntry)) {
+        // initialize additions array as empty if it doesn't already exist
+        timelineEntry.additions ||= [];
+        enbs.populateManualInputs(timelineEntry, timelineEntry.nextEntry, enbs.SINGLE_KEY,
             manualResultMap[enbs.SINGLE_KEY]);
-        trip.finalInference = {};
+        timelineEntry.finalInference = {};
     } else {
-        console.log("Trip information not yet bound, skipping fill");
+        console.log("timelineEntry information not yet bound, skipping fill");
     }
   }
 
   /**
-   * Embed 'inputType' to the trip
+   * Embed 'inputType' to the timelineEntry
    * This is the version that is called from the list, which focuses only on
    * manual inputs. It also sets some additional values 
    */
-  enbs.populateManualInputs = function (trip, nextTrip, inputType, inputList) {
+  enbs.populateManualInputs = function (timelineEntry, nextTimelineEntry, inputType, inputList) {
       // Check unprocessed labels first since they are more recent
-      const unprocessedLabelEntry = InputMatcher.getAdditionsForTrip(trip, inputList);
+      const unprocessedLabelEntry = InputMatcher.getAdditionsForTimelineEntry(timelineEntry, inputList);
       var userInputEntry = unprocessedLabelEntry;
-      if (userInputEntry.length == 0) {
-          userInputEntry = trip.trip_addition;
+      if (!angular.isDefined(userInputEntry)) {
+          userInputEntry = timelineEntry.timelineEntry_addition[enbs.inputType2retKey(inputType)];
```

Comment:
Where is the property `timelineEntry_addition` coming from? I do not see any other references to this object when searching the e-mission-phone:places-survey code. Is it coming from the server?

Reply:
It was intended to be coming from the server, but now it's just called "additions" so it's coming from nowhere. I'll fix this soon

Reply:
fixed 8b2ef77e459bc381e0660c10ef36bd1fad7db512